superqt/combobox/_enum_combobox.py

```diff
@@ -0,0 +1,84 @@
+from typing import Optional, Type, TypeVar, Union
+from enum import Enum
+
+from ..qtcompat.QtWidgets import QComboBox
+from ..qtcompat.QtCore import Signal, PYQT5
+
+EnumType = TypeVar("EnumType", bound=Enum)
+enum_type = Enum if PYQT5 else object
+
+def _get_name(enum_value: Enum):
+    """Create human readable name if user does not provide own implementation of __str__"""
+    if enum_value.__str__.__module__ != "enum":
+        # check if function was overloaded
+        name = str(enum_value)
+    else:
+        name = enum_value.name.replace("_", " ")
+    return name
+
+
+class EnumComboBox(QComboBox):
+    """
+    This is implementation of variant of EnumComboBox for select enum value.
```

Comment:
place comment on first line, with [empty line between it and description](https://www.python.org/dev/peps/pep-0257/#id17).

Reply:
I do not understand. Maybe I should write docstring from scratch?

Reply:
crap... sorry, looks like I commented on an old commit.  here was my suggestion

```
    """ComboBox presenting options from a python Enum.

    If the Enum class does not implement `__str__` then a human readable name
    is created from the name of the enum member, replacing underscores with spaces.
    """
```

Reply:
changed